docs/sources/api/_index.md

```diff
@@ -479,7 +495,11 @@ $ curl -G -s  "http://localhost:3100/loki/api/v1/label/foo/values" | jq
 }
 ```
 
-## `GET /loki/api/v1/tail`
+## Stream log messages
```

Comment:
hmmm by renaming this section, the href present in line 33 isn't going to work. Do you mind fixing it? The same applies for the other subheader changes.